### PR TITLE
[BULK] - DocuTune - Rebranding of Azure Active Directory to Microsoft Entra terminology

### PR DESCRIPTION
**Pull request by alexbuckgit**
Applying Azure AD rebranding changes as announced on July 11, 2023 and outlined in https://aka.ms/AzureADNewName. Changes are part of the Microsoft-wide rebranding effort.

CorrelationId: 2520d58e-25ff-469e-af14-2834d2a1505e
PointOfContact: @CelesteDG

#docutune

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: msal-javascript-conceptual/browser/resources-and-scopes.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ms.reviewer: dmwendia, cwerner, owenrichards, kengaderdus
 
 # Resources and scopes
 
-Azure Active Directory v2.0 & Microsoft Identity Platform employs a *scope-centric* model to access resources. Here, a *resource* refers to any application that can be a recipient of an **Access Token** (such as [MS Graph API](/graph/overview) or your own web API), and a *scope* (*aka* "permission") refers to any aspect of a resource that an **Access Token** grants rights.
+Azure Active Directory v2.0 & Microsoft identity platform employs a *scope-centric* model to access resources. Here, a *resource* refers to any application that can be a recipient of an **Access Token** (such as [MS Graph API](/graph/overview) or your own web API), and a *scope* (*aka* "permission") refers to any aspect of a resource that an **Access Token** grants rights.
```
Comment (Dickson-Mwendia) [inline msal-javascript-conceptual/browser/resources-and-scopes.md:15]:
```suggestion
Microsoft identity platform employs a *scope-centric* model to access resources. Here, a *resource* refers to any application that can be a recipient of an **Access Token** (such as [MS Graph API](/graph/overview) or your own web API), and a *scope* (*aka* "permission") refers to any aspect of a resource that an **Access Token** grants rights.
```

File: msal-javascript-conceptual/browser/working-with-b2c.md [hunk 2]
```diff
@@ -80,21 +80,23 @@ const tokenRequest = {
...
 ```
 
-## AAD vs B2C Endpoints
+<a name='aad-vs-b2c-endpoints'></a>
 
-A major difference between **Azure AD** (*AAD* for short) vs. **B2C** tenants are with respect to their **endpoints**.
+## Microsoft Entra vs B2C Endpoints
 
-An **AAD** tenant:
+A major difference between **Microsoft Entra ID** (*Microsoft Entra ID* for short) vs. **B2C** tenants are with respect to their **endpoints**.
```
Comment (Dickson-Mwendia) [inline msal-javascript-conceptual/browser/working-with-b2c.md:87]:
```suggestion
A major difference between **Microsoft Entra ID**  and **Azure AD B2C** tenants is their **endpoints**.
```

**Review by Dickson-Mwendia — CHANGES_REQUESTED** [review]
Updates looking great. Suggested a few changes here.

**Pull request merged**